### PR TITLE
Add build_deploy.sh file

### PR DESCRIPTION
**Pull request by mbharatk**
`build_deploy.sh` file to be added for app-interface to build and push github-refresh-cronjob images.

### CODE REVIEW

**Review by sawood14012 — CHANGES_REQUESTED** [review]
The Docker file is using the old images please create a new dockerfile or update the dockerfile with using UBI image as base

**Comment by sawood14012** [comment]
Rest everything looks fine

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.centos.org/centos/centos:7
+FROM registry.access.redhat.com/ubi8/python-36:latest
```
Comment (sawood14012) [inline Dockerfile]: just changing the base image wont work out @mbharatk you might have to manually build and verify or by trial and error method if it works PTAL

**Review by sawood14012 — CHANGES_REQUESTED** [review]

**Review by rafiu007 — COMMENTED** [review]
Please test out the changes before pushing. for this case build out the image and try deploying it.For migration related tasks look at other repos where migration has been done. like api-server,jobs api. Look at those prs.
it will help

**Review by sawood14012 — APPROVED** [review]
LGTM (Please merge if it is working on dev cluster)

**Comment by sawood14012** [comment]
@mbharatk Please take a look at unit tests

File: qa/runtests.sh [hunk 2]
```diff
@@ -16,14 +16,17 @@ check_python_version() {
 
 echo "Create Virtualenv for Python deps ..."
 function prepare_venv() {
+        python3 -m pip install --user virtualenv
```
Comment (sawood14012) [inline qa/runtests.sh]: it is not recommended to install the dependency here as actions already installs it ref: https://github.com/fabric8-analytics/fabric8-analytics-github-refresh-cronjob/blob/master/.github/workflows/ci-build.yml#L40

**Review by sawood14012 — CHANGES_REQUESTED** [review]

**Review by sawood14012 — APPROVED** [review]
LGTM

**Pull request merged**